src/pkg/cli/getServices_test.go

```diff
@@ -2,39 +2,156 @@ package cli
 
 import (
 	"context"
-	"errors"
+	"net/http/httptest"
+	"strings"
 	"testing"
+	"time"
 
-	"github.com/DefangLabs/defang/src/pkg/cli/client"
+	cliClient "github.com/DefangLabs/defang/src/pkg/cli/client"
 	"github.com/DefangLabs/defang/src/pkg/cli/compose"
+	"github.com/DefangLabs/defang/src/pkg/term"
 	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
+	"github.com/DefangLabs/defang/src/protos/io/defang/v1/defangv1connect"
+	"github.com/bufbuild/connect-go"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-type mockGetServicesProvider struct {
-	client.Provider
-}
+var mockCreatedAt = timestamppb.Now()
+var mockExpiresAt = timestamppb.New(time.Now().Add(1 * time.Hour))
```

Comment:
Use a UTC fixed time so you can actually test the formatting by hardcoding the string in the output.